extensions/wikia/HubRssFeed/tests/RssFeedServiceTest.php

```diff
@@ -3,10 +3,8 @@
 class RssFeedServiceTest extends WikiaBaseTest {
 
 	public function setUp() {
-		$dir = dirname( __FILE__ ) . '/../';
-		$this->setupFile = $dir . 'HubRssFeed.setup.php';
-
 		parent::setUp();
+		require_once __DIR__ . '/../RssFeedService.class.php';
```

Comment:
I assume this change was made because of logic performed in `HubRssFeed.setup.php`.